frontend2/src/components/sidebar/index.tsx

```diff
@@ -24,7 +24,7 @@ const Sidebar: React.FC<SidebarProps> = ({ collapsed }) => {
   const linkBase = `/${episodeId}/`;
 
   return collapsed ? null : (
-    <div className="flex flex-col gap-8 py-4 h-full bg-gray-50 shadow-gray-200 shadow-sm">
+    <div className="flex flex-col gap-8 py-4 h-full bg-gray-50 shadow-gray-200 shadow-sm hidden sm:block">
```

Comment:
This div should have display=`flex` on screens larger than 640px, not display`block`. Adding `sm:block` makes the div have display=`block` on screens larger than the `sm` breakpoint (640px).

we should change this className to be:
`"hidden sm:flex flex-col gap-8 py-4 h-full bg-gray-50 shadow-gray-200 shadow-sm"`


here's a useful link to understand how style using breakpoints: https://tailwindcss.com/docs/responsive-design#targeting-mobile-screens